packages/editor/templates/Coin/README.md

```diff
@@ -236,6 +236,15 @@ function isValidAmount(amount) {
 
 And that's all the code. To see the dapp in action, click _Compile_, then _Deploy_ found under the disclosure triangle of the contract file, then open the _Preview_ tab to see the frontend of the dapp.
 
-## Find out more
+## Next Steps
 
 You can read a full tutorial that accompanies this example dapp, plus many more tutorials, on [kauri.io](https://kauri.io/article/bdd65d6155a74b8aa52672b46b7230a8/v1/a-fullstack-dapp-for-creating-tokens).
+
+Other resources useful to continue your dapp development journey are:
+
+- [Remix](https://remix.ethereum.org/): Ethereum IDE and tools for the web
+- [Grid](https://grid.ethereum.org/): Download, configure, and run Ethereum nodes and tools
```

Comment:
Why putting Grid instead of something like Ganache? Especially things like Truffle / Buidler / Embark + Ganache

Reply:
Because it's a EF project too, and it's kind of "the whole stack", but we can certainly add others! More likely Truffle and Ganache, I have no experience of the others and their "beginner friendliness"

Reply:
I agree, in my mind Grid is tool for more advanced users who want to run their own node vs. Ganache which is more beginner-friendly for spinning up a local dev blockchain.

Reply:
@ChrisChinchilla (unrelated to this PR) I recommend you check out Buidler & others to get a sense of the tooling out there.

Reply:
Not the impression I got from them, but I did get a personal demo, maybe their website copy is not helpful.